src/components/page-configuration/index.tsx

```diff
@@ -55,19 +55,31 @@ export interface PageConfigurationProps
   appName: string;
 }
 
+function getConfigBranch(configBranch: string) {
+  return configBranch?.length > 0 ? configBranch : 'master';
+}
```

Comment:
remember to add return typing on methods, can also be simplified
```suggestion
function getConfigBranch(configBranch: string): string {
  return configBranch || 'master';
}
```